.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build Docker image
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
```

Comment:
`contents: read` should be a top level permission. See https://llvm.org/docs/CIBestPractices.html.

Reply:
Done in 9992e48bcdae65788ffd97b0213b6fe6612f7c08